docs/UserGuide.md

```diff
@@ -9,331 +9,424 @@ Interface** (CLI) while still providing an easy way to visualize all events thro
+Adding of events that overlaps in time is not allowed.
+</div>
 
+[[Back to top](#table-of-contents)]
 
-<h3 id="recur">Repeat Existing Event</h3>
+<h3 id="recur">Repeat an existing event `recur`</h3>
 
-Recurs an event from the Scheduler until an end date.
+Repeats an event from the scheduler until the specified end date.
 
-Format: `recur <index> d/<end date> every/<recur factor>`
+Format: `recur <index> d/<end date> every/<recur factor>`   
+Example: `recur 1 d/2024-01-01 every/month`
 
 Parameters:  
-`d/`: Ending date of recurrence for the event in `YYYY-MM-DD` format  
-`every/`: Recur factor to recur the event, given by `day/week/month`
+`d/`: Ending date of recurrence for the event  
+`every/`: Recurring factor to recur the event
+
+<div markdown="span" class="alert alert-info">
+:information_source: **Note:**.  
+Date should follow a `YYYY-MM-DD` format.  
+Recurring factor is given by `day/week/month`.
+</div>
 
 <div markdown="span" class="alert alert-warning">
-:information_source: **Note:**  
+:warning: **Warning:**  
 Event must first exist in the scheduler.  
-Recurring of events not existing in scheduler is not allowed.  
 End date should come after current event date.  
+Recurring of events not existing in scheduler is not allowed.
 </div>
 
-Example: `recur 1 d/2024-01-01 every/month`
+[[Back to top](#table-of-contents)]
 
-[[Back to top](#index)]
+<h3 id="edit">Edit an event `edit`</h3>
 
+Edits an event in the scheduler.
 
-<h3 id="edit">Edit An Event</h3>
-
-Edits an event in the Scheduler.
-
-Format: `edit <index> n/<event> d/<date> s/<start time> e/<end time>`
+Format: `edit <index> n/<event> d/<date> s/<start time> e/<end time>`  
+Example: `edit 1 n/tennis` or `edit 3 d/2023-06-01 s/10:00`
 
 Parameters:  
-`n/`: Name or description of event  
+`n/`: Name of the event  
 `d/`: Date of the event  
 `s/`: Start time of the event  
 `e/`: End time of the event
 
+<div markdown="span" class="alert alert-info">
+:information_source: **Note:**  
+Name is alphanumeric only.  
+Date should follow a `YYYY-MM-DD` format.  
+Time should be in 24-hour `HH:mm` format, ranging from `00:00` to `23:59`. Leading zero are required.
+</div>
+
 <div markdown="span" class="alert alert-warning">
 :warning: **Warning:** At least one field must be provided.
 </div>
 
-Example: `edit 1 n/tennis` or `edit 3 d/2023-06-01 s/10:00`
-
-[[Back to top](#index)]
-
+[[Back to top](#table-of-contents)]
 
-<h3 id="delete">Delete Event</h3>
+<h3 id="delete">Delete events `delete`</h3>
 
-Deletes the event(s) in the Scheduler.
-
-Format: `delete <index>` or `delete <index index ... index>`
+Deletes the event(s) in the scheduler.
 
+Format: `delete <index>` or `delete <index index ... index>`  
 Example: `delete 1` or `delete 1 4 5`
 
-[[Back to top](#index)]
+<div markdown="span" class="alert alert-warning">
+:warning: **Warning:** Index must be a positive integer and exist in the scheduler.
+</div>
 
+[[Back to top](#table-of-contents)]
 
```

Comment:
I used 2 blank lines between `Back to top` and the next header, to increase readability for us. Perhaps do the same also?